schema/revert/tables/project_revision_002_set_amendment_statuses.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert cif:tables/project_revision_002_set_amendment_statuses from pg
+
+begin;
+
+alter table cif.project_revision alter column amendment_status drop default;
```

Comment:
I assume I don't want to also revert setting the statuses?

Reply:
I can't think of a good reason to revert statuses